src/main/java/com/iota/iri/service/TipsSolidifier.java

```diff
@@ -30,9 +30,15 @@ public TipsSolidifier(final Tangle tangle,
     public void init() {
         solidityRescanHandle = new Thread(() -> {
 
+            long lastTime = 0;
             while (!shuttingDown) {
                 try {
                     scanTipsForSolidity();
+                    long now = System.currentTimeMillis();
+                    if ((now - lastTime) > 10000L) {
+                        lastTime = now;
+                        log.debug("#Solid/NonSolid: {}/{}", tipsViewModel.solidSize(), tipsViewModel.nonSolidSize());
```

Comment:
Can you also write this to ZMQ? Use the following topic "debug-TipsSolidifier" we want to change our ZMQ topics in the future anyhows.

Reply:
@gjeee after thinking about this farther this change shouldn't be part of this PR.
I wanted to use the ZMQ for testing but it should be done as part of this issue:
https://github.com/iotaledger/iri/issues/741.

Maybe in order to create a test we will add a ZMQ message separate from #741 but let's not do it as part of this PR.

So you have only 1 change.

Reply:
roger.